hazelcast/src/main/java/com/hazelcast/query/impl/IndexImpl.java

```diff
@@ -79,7 +86,7 @@ public void clear() {
     }
 
     @Override
-    public void saveEntryIndex(QueryableEntry e) throws QueryException {
+    public void saveEntryIndex(QueryableEntry e, Object oldRecordValue) throws QueryException {
```

Comment:
What does old mean? Because it makes it sound if there also is a current record value. So we we need the prefix 'old' at all?